systemtests/src/test/java/io/enmasse/systemtest/plans/PlansTest.java

```diff
@@ -153,6 +157,70 @@ public void testQuotaLimits() throws Exception {
                 ), username, password);
 
+        //check broker limits
+        checkLimits(addressSpace,
+                Arrays.asList(
+                        Destination.queue("q1", queuePlan.getName()),
+                        Destination.queue("q2", queuePlan.getName())
+                ),
+                Collections.singletonList(
+                        Destination.queue("q3", queuePlan.getName())
+                ), username, password);
+
+        checkLimits(addressSpace,
+                Arrays.asList(
+                        Destination.queue("q1", queuePlan.getName()),
+                        Destination.queue("q2", queuePlan.getName()),
+                        Destination.queue("q3", queuePlan2.getName()),
+                        Destination.queue("q4", queuePlan2.getName()),
+                        Destination.queue("q5", queuePlan2.getName()),
+                        Destination.queue("q6", queuePlan2.getName())
+                ), null, username, password);
+
+        //check aggregate limits
+        checkLimits(addressSpace,
+                Arrays.asList(
+                        Destination.topic("t1", topicPlan.getName()),
+                        Destination.topic("t2", topicPlan.getName())
+                ),
+                Collections.singletonList(
+                        Destination.topic("t3", topicPlan.getName())
+                ), username, password);
+    }
+
+    @Test
+    public void testQuotaLimitsSharded() throws Exception {
+        String username = "quota_user";
+        String password = "quotaPa55";
+        //define and create address plans
+        AddressPlan queuePlan = new AddressPlan("queue-sharded-test1", AddressType.QUEUE,
+                Collections.singletonList(new AddressResource("broker", 1.0)));
+
+        AddressPlan topicPlan = new AddressPlan("topic-sharded-test2", AddressType.TOPIC,
+                Arrays.asList(
+                        new AddressResource("broker", 1.0),
+                        new AddressResource("router", 0.01)));
+
+        plansProvider.createAddressPlanConfig(queuePlan);
+        plansProvider.createAddressPlanConfig(topicPlan);
+
+        //define and create address space plan
+        List<AddressSpaceResource> resources = Arrays.asList(
+                new AddressSpaceResource("broker", 0.0, 2.0),
+                new AddressSpaceResource("router", 1.0, 2.0),
+                new AddressSpaceResource("aggregate", 0.0, 3.0));
+        List<AddressPlan> addressPlans = Arrays.asList(queuePlan, topicPlan);
+        AddressSpacePlan addressSpacePlan = new AddressSpacePlan("test-quota-limits-sharded", "test-quota-limits-sharded",
+                "standard-space", AddressSpaceType.STANDARD, resources, addressPlans);
+        plansProvider.createAddressSpacePlanConfig(addressSpacePlan);
+
+        //create address space with new plan
+        AddressSpace addressSpace = new AddressSpace("test-quota-limits-sharded", AddressSpaceType.STANDARD,
```

Comment:
could be better to use different name for address-space instance and address-space-plan name. You could avoid some irregularity in the future debugging. (if some trouble happen); again my personal opinion